### PR TITLE
Bundled With MicroBundle

### PR DESCRIPTION
**Pull request by nicko-winner**
Package size is now reasonably small.

It likely is possible to get this working with Parcel especially v2, and it would decrease build times, but this micro bundle project seems to centered around making bundles smaller, which might be more ideal in our case since we do not suffer from long build times given the nature and size of this project. Parcel did not have very good docs that I could find around building libraries.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
-    "dist"
-  ],
+  "source": "src/index.mjs",
+  "main": "dist/helix-react.js",
+  "umd:main": "dist/helix-react.umd.js",
+  "module": "dist/helix-react.m.js",
   "scripts": {
     "install:clean": "rm -rf node_modules && yarn install",
-    "build": "parcel build src/index.mjs",
+    "build": "microbundle -i src/index.mjs --external react,react-dom --name HelixReact --jsx React.createElement --jsxFragment React.Fragment --compress false --sourcemap false",
```
Comment (nicko-winner) [inline package.json:11]: When your build tool replaces a config file with arguments does that make it a zero config build solution? (I guess so)

I went this route because of this thread:
https://github.com/parcel-bundler/parcel/issues/504
https://github.com/parcel-bundler/parcel/issues/192

File: package.json [hunk 2]
```diff
@@ -55,15 +53,18 @@
...
+    "use-onclickoutside": "^0.3.1",
+    "prop-types": "^15.7.2"
   },
-  "dependencies": {
-    "classnames": "^2.2.6",
-    "use-onclickoutside": "^0.3.1"
+  "peerDependencies": {
+    "helix-ui": "^0.23.0",
+    "prop-types": "15.x",
+    "react": "^16.6.3"
```
Comment (nicko-winner) [inline package.json:67]: These are all excluded from bundles

Comment (100stacks) [reply package.json]: ok makes sense

File: package.json [hunk 3]
```diff
@@ -1,16 +1,14 @@
...
-  "unpkg": "dist/index.js",
-  "files": [
-    "dist"
-  ],
+  "umd:main": "dist/index.umd.js",
+  "module": "dist/index.m.js",
   "scripts": {
     "install:clean": "rm -rf node_modules && yarn install",
-    "build": "parcel build src/index.mjs",
+    "build": "microbundle -i src/index.mjs --external react,react-dom,classnames,use-onclickoutside --name HelixReact --jsx React.createElement --jsxFragment React.Fragment --compress false --sourcemap false",
```
Comment (nicko-winner) [inline package.json:11]: I for sure did not want to remove all the React from the bundles, but I think i was going overkill taking classnames, and user-onclickoutside out of the bundles. I think those have be stay in there.
```suggestion
    "build": "microbundle -i src/index.mjs --external react,react-dom --name HelixReact --jsx React.createElement --jsxFragment React.Fragment --compress false --sourcemap false",
```

File: package.json [hunk 4]
```diff
@@ -1,16 +1,14 @@
 {
   "name": "@helix-design-system/helix-react",
-  "version": "1.0.0-alpha.4",
+  "version": "1.0.0-alpha.5",
```
Comment (100stacks) [inline package.json:3]: You can revert this change.  The version will get bumped when we publish the next npm version.

File: package.json (hunk 1 above)
Comment (100stacks) [reply package.json:11]: Ok, sounds good.  I'm up for trying out this bundler.

File: package.json (hunk 4 above)
Comment (nicko-winner) [reply package.json:3]: Cool. Some time when your free id like to learn more about how the release part happens.
```suggestion
  "version": "1.0.0-alpha.4",
```
